qa/workunits/rbd/cli_migration.sh

```diff
@@ -89,7 +112,17 @@ compare_images() {
     local dst_image=$2
     local ret=0
 
-    export_raw_image ${dst_image}
+   # Export both if image is from a namespace
```

Comment:
There is another thread on this, commenting here just in case: I'm still not convinced that `compare_images()` should ever (re-)export the source image.